db/migrate/20230830115220_validate_create_doorkeeper_openid_connect_tables.rb

```diff
@@ -0,0 +1,5 @@
+class ValidateCreateDoorkeeperOpenidConnectTables < ActiveRecord::Migration[7.0]
```

Comment:
Why is this a separate migration? Can we not just include it in the previous migration that adds the tables?

Reply:
Hm, db:migrate was failing due to possible deadlock when creating foreign keys and validating them in one shot, and suggested splitting to two migration scripts, which I did. I guess I could merge them into one.

Here is the error message from db:migrate

> === Dangerous operation detected #strong_migrations ===
>
> Adding a foreign key blocks writes on both tables. Instead,
> add the foreign key without validating existing rows,
> then validate them in a separate migration.

Reply:
If there's a good reason then fine.

Reply:
Added clarifying comments in migration scripts.